en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,62 @@
+Customize Content Attribute Display
+===================================
+
+Context
+-------
+
+In order to have minimum constraints on contents, Open Orchestra allows you to store any type
+of ``value`` in the ``ContentAttribute``.
+Sometimes, there is no easy way to render the value to the backoffice user.
+For instance, to list entities with a field type like ``choice``, that can return an array based
+on the field options, deal with array like a string returns the `array` string on display for all
+field entries.
+So the ``ContentAttribute`` class provide a specific attribute ``stringValue`` to represent the
```

Comment:
So is, imho, a little to direct to get the conclusion. I would have said `In that way`